lib/std/os/linux.zig

```diff
@@ -1621,6 +1621,10 @@ pub fn tcsetattr(fd: fd_t, optional_action: TCSA, termios_p: *const termios) usi
     return syscall3(.ioctl, @bitCast(usize, @as(isize, fd)), T.CSETS + @enumToInt(optional_action), @ptrToInt(termios_p));
 }
 
+pub fn tcsetpgrp(fd: fd_t, pgrp: *pid_t) usize {
```

Comment:
see here

Reply:
oh, no i did not, oops. i'll update this and remove that, i'm not sure how it got there.